1-js/06-advanced-functions/12-arrow-functions/article.md

```diff
@@ -61,28 +61,27 @@ let group = {
 group.showList();
 ```
 
-The error occurs because `forEach` runs functions with `this=undefined` by default, so the attempt to access `undefined.title` is made.
+এক্ষেত্রে `forEach` এর ফাংশনটির ডিফল্ট কন্টেক্স হত `this=undefined`, সুতরাং এটি প্রপার্টিকে এভাবে অ্যাক্সেস করতে চাইত `undefined.title`।
 
-That doesn't affect arrow functions, because they just don't have `this`.
+যেহেতু অ্যারো ফাংশনের `this` নেই, তাই এটি কোন এরর রিটার্ন করবে না।
 
-```warn header="Arrow functions can't run with `new`"
-Not having `this` naturally means another limitation: arrow functions can't be used as constructors. They can't be called with `new`.
+```warn header="`new` এ সাথে অ্যারো ফাংশন রান করে না"
+`this` না থাকার কারণে এর একটি লিমিটেশন আছে: অ্যারো ফাংশনকে আমরা *constructors* হিসেবে ব্যবহার করতে পারব না। কেননা এরা `new` দ্বারা কল হয় না।
 ```
 
 ```smart header="Arrow functions VS bind"
-There's a subtle difference between an arrow function `=>` and a regular function called with `.bind(this)`:
+অ্যারো ফাংশন `=>` এবং ্রেগুলার ফাংশন `.bind(this)` এর মাঝে একটি পার্থক্য আছে:
 
 - `.bind(this)` creates a "bound version" of the function.
 - The arrow `=>` doesn't create any binding. The function simply doesn't have `this`. The lookup of `this` is made exactly the same way as a regular variable search: in the outer lexical environment.
 ```
 
-## Arrows have no "arguments"
+## অ্যারো ফাংশনের "arguments" ভ্যারিয়েবল
 
-Arrow functions also have no `arguments` variable.
+অ্যারো ফাংশনে `arguments` ভ্যারিয়েবল নাই।
 
-That's great for decorators, when we need to forward a call with the current `this` and `arguments`.
-
-For instance, `defer(f, ms)` gets a function and returns a wrapper around it that delays the call by `ms` milliseconds:
+ডেকোরেটার্সের জন্য এটি দারুণ, যখন আমাদের  `this` এবং `arguments` এর সাহায্যে একটি ফরওয়ার্ড কল করতে হয়।
```

Comment:
Please dont mention `decorators` as `ডেকোরেটার্স` rather use `ডেকোরেটর ফাংশন` or in plurals `ডেকোরেটর ফাংশন গুলোর`